templates/jobs/job-create-indices.yaml

```diff
@@ -71,6 +71,8 @@ spec:
               value: {{ include "graphdb.fullname.service.headless" . }}
             - name: GRAPHDB_SERVICE_PORT
               value: {{ .Values.headlessService.ports.http | quote }}
+            - name: GRAPHDB_PROTOCOL
```

Comment:
`function createIndicesFromFiles()` needs to be updated to read this.

Reply:
The issue was with the https://github.com/Ontotext-AD/graphdb-helm/blob/fix-issue-174/files/scripts/graphdb.sh#L165 waitNodes function that was called within but I changed the createIndices to rely on the protocol, too.